llpc/test/shaderdb/OOB_Check_Load_Array_lit.frag

```diff
@@ -0,0 +1,36 @@
+// This test checks if load access to a 2D (multidimensional) array using a runtime index
+// will be transformed to a out of bounds checks against all bounds of the array, moving the load into a conditionally executed BB.
+// The OOB check will possibly zero out the load results.
+
+// BEGIN_SHADERTEST
+/*
+; RUN: amdllpc -spvgen-dir=%spvgendir% -v %gfxip %s -enable-scratch-bounds-checks | FileCheck -check-prefix=SHADERTEST %s
+; SHADERTEST-LABEL: {{^// LLPC}} SPIRV-to-LLVM translation results
+; SHADERTEST: alloca [2048 x [2 x <4 x float>]], align 16, addrspace(5)
```

Comment:
Would it be possible to match against the name of this instruction in subsequent checks?